README.md

```diff
@@ -3,6 +3,9 @@ ilib-loctool-webos-ts-resource is a plugin for the loctool that
 allows it to read and localize ts resource files. This plugins is optimized for webOS platform.
 
 ## Release Notes
+v1.1.0
+* Fixed an issue which doesn't not generated `key` value.
```

Comment:
do you mean "which does" or "which does not" ?

Reply:
I meant `which does not`. Currently, the TS file doesn't include the `key` value when it exists. It should be written under the `comment` tag in TS file. I update the readme file a bit.